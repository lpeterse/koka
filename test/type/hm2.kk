// fun choose1(x:a,y:a) { x }

fun choose2(x:a,y:a) : a { x }

fun choose3(x,y) { if (True) then x else y }
