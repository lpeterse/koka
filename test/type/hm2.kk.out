choose2     : forall<a> (x : a, y : a) -> a
choose3     : forall<a> (x : a, y : a) -> a
